### PR TITLE
Kma/feature branch example

### PR DESCRIPTION
**Pull request by agesak**

### CODE REVIEW

File: fav-color.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+BLUE!!!!!!!!
```
Comment (lina5147) [inline fav-color.txt:1]: Cool! My favorite color is yellow!

**Review by lina5147 — CHANGES_REQUESTED** [review]
add more content to read me page

File: fav-color.txt (hunk 1 above)
Comment (agesak) [reply fav-color.txt:1]: yes ma'am

Comment (agesak) [reply fav-color.txt:1]: @lina5147 is this better now?

**Review by lina5147 — APPROVED** [review]

**Pull request merged**